usaspending_api/search/filters/postgres/defc.py

```diff
@@ -66,7 +66,7 @@ def build_def_codes_filter(cls, filter_values):
 
             results = cursor.fetchall()
 
-        results = [result[0] for result in results]
+        results = {result[0] for result in results}
```

Comment:
Would really like to see a comment on this explaining why we're using a set (to prevent duplicates)

Reply:
Added a comment explaining what the raw SQL is doing and why there can be duplicates which need to be removed.